wind.go

```diff
@@ -336,6 +339,9 @@ func (w *Window) unlock1() {
 
 // Unlock releases the lock on each clone of w
 func (w *Window) Unlock() {
+	if w.body.file == nil {
+		return
```

Comment:
Shouldn't we call `w.unlock1()` here before returning? We need undo these lines from the `Lock` method:
```go
w.lk.Lock()
w.ref.Inc()
w.owner = owner
```

Reply:
Yes, you're right.